numpy/testing/utils.py

```diff
@@ -28,11 +29,28 @@
            'raises', 'rand', 'rundocs', 'runstring', 'verbose', 'measure',
            'assert_', 'assert_array_almost_equal_nulp', 'assert_raises_regex',
            'assert_array_max_ulp', 'assert_warns', 'assert_no_warnings',
-           'assert_allclose', 'IgnoreException', 'clear_and_catch_warnings']
+           'assert_allclose', 'IgnoreException', 'clear_and_catch_warnings',
+           'SkipTest', 'KnownFailure']
 
 
-verbose = 0
+class KnownFailure(Exception):
+    '''Raise this exception to mark a test as a known failing test.'''
+    pass
+
 
+# nose.SkipTest is unittest.case.SkipTest
+# import it into the namespace, so that it's available as np.testing.SkipTest
+try:
+    from unittest.case import SkipTest
+except ImportError:
+    # on py2.6 unittest.case is not available. Ask nose for a replacement.
+    def _get_skip():
```

Comment:
As an attempt to avoid a hard dependency on nose, parroted from other functions in this module (eg https://github.com/numpy/numpy/blob/master/numpy/testing/utils.py#L1120).
AFAICS, `import_nose` either returns the `nose` module or `raises ImportError`.